Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -576,13 +576,7 @@ YUI.add('ez-locationmodel-tests', function (Y) {
                 }
             });
 
-            this.model.loadPath({
-                api: this.capiMock
-            }, function (error, locations) {
-                Assert.isFalse(
-                    error,
-                    "No error should be detected"
-                );
+            this._assertLocations = function (locations) {
```

Comment:
that'll work but that's a bit weird, define a proper method or put the function in the local variable